scimath/units/geo_units.py

```diff
@@ -10,7 +10,12 @@
 
 from copy import copy
 
-from scimath.units.SI import *
+from scimath.units.SI import ampere, atto, becquerel, candela, centi, \
```

Comment:
Let's import `unit`, `dimensionless` and `none` from `scimath.units.unit` (the module where they're defined) instead of from `scimath.units.SI`.